cmake/FindCompiler.cmake

```diff
@@ -25,10 +25,6 @@ include(FindPackageHandleStandardArgs)
 check_cxx_compiler_flag("-fsycl" is_dpcpp)
 
 if(is_dpcpp)
-  find_package_handle_standard_args(SYCL DEFAULT_MSG)
```

Comment:
Do you know what exactly this part does and why we don't need it anymore?

Reply:
No, we couldn't exactly find the purpose of this code. However, @AlexReinemann tested this change with DPC++ LLVM compiler and oneAPI DPC++ compiler and it works as expected.

As far as I understand, checking for "-fsycl" should suffice as an indicator that the compiler is supported. Do you think we need anything beyond this check?

Reply:
I believe this set the variable SYCL and was not working correctly.  since the if statement below was changed to not use the variable SYCL, then this call is no longer needed either.